src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -773,7 +773,13 @@
 helps['role assignment list'] = """
 type: command
 short-summary: List role assignments.
-long-summary: By default, only assignments scoped to subscription will be displayed. To view assignments scoped by resource or group, use `--all`.
+long-summary: >-
+    By default, only assignments scoped to subscription will be displayed. To view assignments scoped by resource or group, use `--all`.
+
+
+    [WARNING] Classic administrators will be deprecated on August 31, 2024.
+    After August 31, 2024, all classic administrators risk losing access to the subscription.
+    Delete Classic Admins who no longer need access or assign an Azure RBAC role for fine-grained access control.
```

Comment:
This message is copied from Azure Portal's banner (https://github.com/Azure/azure-cli/issues/27199#issuecomment-2230388518). However, the banner cannot render correctly today:

![image](https://github.com/user-attachments/assets/81bf661a-a336-4329-8203-6ef76ee832dd)

I don't know what is inside that "Learn more" hyperlink.

Reply:
I used Inspect tool to find the link to be https://go.microsoft.com/fwlink/?linkid=2238474

![image](https://github.com/user-attachments/assets/d3d5e5e2-376b-4b48-b724-a3e1b8520f81)